draft-irtf-cfrg-vdaf.md

```diff
@@ -1452,7 +1440,7 @@ The parameters provided by a concrete FLP are listed in {{flp-param}}.
 | `VERIFIER_LEN`   | Length of the verifier message generated by querying the input and proof |
 | `Measurement`    | Type of the measurement   |
 | `AggResult`      | Type of the aggregate result |
-| `Field`          | As defined in ({{field}}) |
+| `Field`          | As defined in ({{fields}}) |
```

Comment:
This should point to `#field` as well, as that's where the class is defined.

Reply:
Whoopsies, this is a regex whoopsy daisy! Thanks